### DISCUSSION TITLE
Suggestion for new supernova module

### DISCUSSION THREAD

**Discussion by itrharrison** [skypyproject/skypy#530] [Infrastructure]
In order to allow the work from #140 to be preserved, I would like to reopen the PR with the target being a supernova module. This would then also provide a target for any further development of supernova models (which seem an important component to cosmology).

This would involve starting a new branch called `module/supernova`

**Comment by Lucia-Fonseca** [comment]
Is the model in this PR worth bringing it?
Is worth creating a module just for this feature?

**Reply by itrharrison** [reply]
I think there are a couple of reasons it is worthwhile doing (assuming we do not want to throw away #140 completely):
- It is better to have PRs correct (targeting a module branch) rather than incorrect (targeting main) regardless of whether they are merge-able. Any new contributor clicking around for examples of what to do should be presented with good examples not bad ones.
- A broader point about increasing the visible breadth of SkyPy: it would be nice to have more even simple modules within the package to act as a hook for further contributions. At the moment if one of us tells someone about the concept of SkyPy and they are excited and go take a look at the repo... all they see are galaxies, meaning they may be put off. Versus if they see a (small) ecosystem of different (even very simple) modules which they could possibly contribute to even if they aren't galaxies people.
